### DISCUSSION TITLE
Serve Images

### DISCUSSION THREAD

**Discussion by s-ashwin** [deta/cloud-docs#254] [Help]
Hi, I am trying to serve an image with an endpoint.
But i am not able to retrive the image on hitting the URL.

Try hitting [https://ats-api.deta.dev/data/photo/16](https://ats-api.deta.dev/data/photo/16)

<img width="799" alt="image" src="https://user-images.githubusercontent.com/47049152/172181111-ebf4fa2f-ba3f-4a6d-8e2e-16212705aadb.png">

Code:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/47049152/172181273-25febd6e-978b-48e9-b97d-2a82e80ced4c.png">

Response:
<img width="1306" alt="image" src="https://user-images.githubusercontent.com/47049152/172183695-3321f5ba-7d8a-40d8-9bb6-f35bad83bb77.png">
P.S. I am actually sending the buffer

But this works fine when running the app on local.

**Comment by rohanshiva** [comment]
Can you please take a look at [this](https://docs.deta.sh/docs/drive/node_tutorial) and see if it helps you?

**Reply by s-ashwin** [reply]
I am not actually using deta drive. I use a MySQL Database hosted on another server

**Reply by abdelhai** [reply]
could you provide a minimal version of your code so we could do an end to end reproduction?
